source/manual/alerts/high-priority-tests.html.md

```diff
@@ -20,5 +22,20 @@ $ fab $environment class:monitoring app.reload:nginx
 $ fab $environment class:monitoring app.restart:smokey-loop
 ```
 
+### `Traceback (most recent call last):`
+
+If you see this error in Icinga, it may imply that the `smokey-loop` process has died. You can try looking through the
```

Comment:
I think "may mean" is more straightforward than "may imply"

Reply:
Amended :)